stdlib/public/core/PastEndIndex.swift

```diff
@@ -0,0 +1,60 @@
+//===--- PastEndIndex.swift -----------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2018 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+// FIXME: docs
+public enum PastEndIndex<Bound> {
```

Comment:
This type should be @_fixed_layout

Reply:
Aw, and I have to rebase my `@_frozen` PR again. ;-)